source/uri-options/uri-options.rst

```diff
@@ -3,25 +3,25 @@ URI Options Specification
 =========================
 
 :Spec Title: URI Options Specification
-:Spec Version: 1.4.0
+:Spec Version: 1.5.0
 :Author: Sam Rossi
 :Spec Lead: Bernie Hackett
 :Advisory Group: Scott L'Hommedieu
 :Approver(s): Cailin Nelson, Jeff Yemin, Matt Broadstone, Dan Pasette, Prashant Mital, Spencer Jackson
 :Informed: drivers@
 :Status: Accepted (Could be Draft, Accepted, Rejected, Final, or Replaced)
 :Type: Standards
-:Last Modified: 2019-09-08
+:Last Modified: 2020-2-13
 
 
 **Abstract**
 ------------
 
 Historically, URI options have been defined in individual specs, and
 drivers have defined any additional options independently of one another.
-Because of the frustration due to there not being a single place where 
+Because of the frustration due to there not being a single place where
```

Comment:
In the future, submitting all the unrelated whitespace changes as a separate pull request would make this easier to review :)

Reply:
Ah I agree, sorry! I wanted to get feedback on this before I left for long weekend. 
I can revert the whitespace changes if you'd like

Reply:
It doesn't matter now; it just would have saved me a minute or to figuring out if there were any changes here I actually had to review :)